pushapkscript/script.py

```diff
@@ -86,7 +86,10 @@ def main(name=None, config_path=None):
         except ScriptWorkerTaskException as exc:
             traceback.print_exc()
             sys.exit(exc.exit_code)
-    loop.close()
+
+    if close_loop:
+        # Loop cannot be reopen once closed. Not closing it allows to run several tests on main()
```

Comment:
Sounds good.  There may be a better way to handle event loops, but the above is how I've gotten it to work.